app/models/courses_namespaces.rb

```diff
@@ -0,0 +1,3 @@
+class CoursesNamespaces < ApplicationRecord
```

Comment:
The more I review this, the more I think this object needs a different name. It's fundamentally a record that specifies both a wiki and a namespace for a particular course (not just a namespace for the course).

Maybe `CourseWikiNamespaces` or even `TrackedNamespaces`?

The idea is that this class (like all of the ActiveRecord classes in this app) is not just an interface for database data, but it's also a tool for helping us think clearly about the data, packaging up a set of data or relationships between data so that we don't have to think about more specific data structures (eg, an array of wiki ids and namespace integers) when we can avoid it.

Reply:
okay, I think `CourseWikiNamespaces` is good, should I go ahead with it?

Reply:
sure